docs/Guides/Detecting-When-Clients-Abort.md

```diff
@@ -0,0 +1,180 @@
+<h1 align="center">Fastify</h1>
+
+# Detecting When Clients Abort
+
+## Introduction
+
+Fastify provides request events to trigger at certain points in a request's 
+lifecycle. However, there isn't a mechanism built-in to 
+detect unintentional client disconnection scenarios such as when the client's 
+internet connection is interrupted. This guide covers methods to detect if
+and when a client intentionally aborts a request.
+
+Keep in mind, Fastify's clientErrorHandler is not designed to detect when a 
+client aborts a request. This works in the same way as the standard Node HTTP 
+module, which triggers the clientError event when there is a bad request or 
+exceedingly large header data. When a client aborts a request, there is no 
+error on the socket and the clientErrorHandler will not be triggered.
+
+## Solution
+
+### Overview
+
+The proposed solution is a possible way of detecting when a client 
+intentionally aborts a request, such as when a browser is closed or the HTTP 
+request is aborted from your client application. If there is an error in your 
+application code that results in the server crashing, you may require 
+additional logic to avoid a false abort detection.
+
+The goal here is to detect when a client intentionally aborts a connection 
+so your application logic can proceed accordingly. This can be useful for 
+logging purposes or halting business logic.
+
+### Hands-on
+
+For this sample solution, we'll be using the following:
+
+- `node.js v18.12.1`
+- `npm 8.19.2`
+- `fastify 4.11.0`
+
+Say we have the following base server set up:
+
+```js
+import Fastify from 'fastify';
+
+const sleep = async (time) => {
+  return await new Promise(resolve => setTimeout(resolve, time || 1000));
+}
+
+const app = Fastify({
+  logger: {
+    transport: {
+      target: 'pino-pretty',
+      options: {
+        translateTime: 'HH:MM:ss Z',
+        ignore: 'pid,hostname',
+      },
+    },
+  },
+})
+
+app.addHook('onRequest', async (request, reply) => {
+  request.raw.on('close', () => {
+    if (request.raw.aborted) {
```

Comment:
I know this has been merged already, but `aborted` has been marked deprecated since `17.0.0` & `16.12.0`. I personally have been using `!writableFinished`.

Reply:
thanks for catching this. would you like to send a pull request to address this?